processor/configuration/src/main/java/com/redhat/service/smartevents/processor/sources/slack/SlackSourceResolver.java

```diff
@@ -0,0 +1,42 @@
+package com.redhat.service.smartevents.processor.sources.slack;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.Map;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+
+import com.redhat.service.smartevents.infra.exceptions.definitions.user.ActionProviderException;
+import com.redhat.service.smartevents.infra.models.gateways.Action;
+import com.redhat.service.smartevents.infra.models.gateways.Source;
+import com.redhat.service.smartevents.processor.GatewayConfiguratorService;
+import com.redhat.service.smartevents.processor.GatewayResolver;
+import com.redhat.service.smartevents.processor.actions.input.InputAction;
+
+@ApplicationScoped
+public class SlackSourceResolver implements SlackSource, GatewayResolver<Source> {
+
+    @Inject
+    GatewayConfiguratorService gatewayConfiguratorService;
+
+    @Override
+    public Action resolve(Source source, String customerId, String bridgeId, String processorId) {
+        Action resolvedAction = new Action();
+        resolvedAction.setType(InputAction.TYPE);
+
+        try {
+            resolvedAction.setParameters(Map.of(
+                    InputAction.ENDPOINT_PARAM, getBridgeWebhookUrl(customerId, bridgeId),
+                    InputAction.CLOUD_EVENT_TYPE, "SlackSource"));
```

Comment:
Reading the code, it is sometimes hard to understand when type is `Slack` and when `SlackSource`